content/en/releases/checklist-for-victims-of-a-data-breach.md

```diff
@@ -0,0 +1,187 @@
+
+Based on current knowledge, Vastaamo's client registry was breached in November 2018 and there has been a second smaller breach in 2019.
+
+You can follow Vastaamo's official updates [at their website](https://vastaamo.fi/ajankohtaista/). (currently only in Finnish)
+
+## Presumably, the stolen data contains the following:
+
+*   First name and surname
+*   Address (Street address, postal code, city, country)
+*   Finnish National Identification Number ("Henkilötunnus")
+*   Phone number
+*   Email address
+*   Written appointment notes, I.e. patient reports
+
+All of the above are classified as [personal information](https://tietosuoja.fi/en/what-is-personal-data). It is possible that your information has not been stolen or that only portions of it have leaked.
+
+## If you suspect you have been a victim of the breach or you have received a personal ransom request, follow these steps:
+
+### Keep a diary of your actions
+
+*   Write down your actions. What you do, when and to whom. For example: _Filed a criminal complaint with the police on 24th of October 2020 at 10:05, requested prohibition of credit from Asiakastieto 24th of October at 5pm._
+*   You can also write down how much time every action took from you.
+*   This information can play a pivotal role in police investigations or when claiming for damages.
+*   Keeping a diary helps you to keep track of what you have done. One's personal memory tends to work poorly under crisis and it is easy to forget things. After a few months, it's hard to recall in what order you took actions without notes.
+
+### Collect evidence
+
+*   If you have received a ransom blackmail, take a screenshot of it and store it on your computer for investigation. Store the original message as well.
+*   Screenshots are important evidence for the authorities. The screenshot should include the sender address.
+*   How to take screenshots in Windows 10: [Instructions from Microsoft](https://support.microsoft.com/en-us/windows/open-snipping-tool-and-take-a-screenshot-a35ac9ff-4a58-24c9-3253-f12bac9f9d44)
+*   How to take screenshots in Apple macOS: [Instructions from Apple](https://support.apple.com/en-us/HT201361)
+*   You can also take a photo of your computer's screen. Make sure that the text is readable from the photo.
+*   Keep receipts from all incurred expenses. For example, services related to credit prohibitions.
+
+### Do not pay the ransom or communicate with the blackmailer
+
+*   The extortion messages contain one's name and identification number. Similar messages have been sent to thousands of other people as well.
+*   Paying the ransom funds organized crime and the criminals may extort you again or alternatively, publish your information despite payment. Do not communicate with the criminal. The information has most likely been leaked to other parties already.
+
```

Comment:
An additional point - people reading this in English are likely to be very unfamiliar with the Finnish Police, so:

There are two points of contact with the authorities: The Finnish Polices and the National Center for Cyber Security. Below are the instructions for contacting both of these authorities.  If you have received a crime number from the Finnish Police then include this number in any communications with both authorities.